src/unknown_00BC20.c

```diff
@@ -819,7 +820,7 @@ s32 func_800113BC() {
 
 GLOBAL_ASM("asm/non_matchings/unknown_00BC20/func_800113CC.s")
 
-void func_80011560(void) {
+s32 func_80011560(void) {
```

Comment:
You sure about that?

Reply:
I believe `func_80033F60` needs this for that function to match. See: https://decomp.me/scratch/viXuF

Reply:
Oh that's fun. In that case can you comment that an undefined return value is expected?